test/e2e/mount_test.go

```diff
@@ -4,7 +4,7 @@
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//      http://www.apache.org/licenses/LICENSE-2.0
+//	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
leave this as spaces instead of tab

Reply:
This change is to fix the format checker error when running make fmtcheck.